docs/source/working_with_aiida/caching.rst

```diff
@@ -4,45 +4,90 @@
 Caching
 *******
 
-When working with AiiDA, you might sometimes re-run calculations which were already successfully executed. Because this can waste a lot of computational resources, you can enable AiiDA to **cache** calculations, which means that it will re-use existing calculations if a calculation with the same inputs is submitted again.
+Enabling caching
+----------------
 
-When a calculation is cached, a copy of the original calculation is created. This copy will keep the input links of the new calculation. The outputs of the original calculation are also copied, and linked to the new calculation. This allows for the new calculation to be a separate Node in the provenance graph and, critically, preserves the acyclicity of the graph.
+There are numerous reasons why you may need to re-run calculations you’ve already done before.
+Since AiiDA stores the full provenance of each calculation, it can *detect* whether a calculation has been run before
+and reuse its outputs without wasting computational resources.
 
-Caching is also implemented for Data nodes. This is not very useful in practice (yet), but is an easy way to show how the caching mechanism works:
+This feature is not (yet) enabled by default. In order to enable hashing:
```

Comment:
Remove the (yet) and perhaps make a note after the fact instead? I.e., make a note block after describing what it is, saying circa when we intend to make it default.

Reply:
I don't think there's an official consensus on this yet... so I'm not sure I can say much more than what I wrote there...

Reply:
Then just remove the (yet) ?

Reply:
Ok, I do it for you ;-)